packages/strapi/lib/middlewares/cron/index.js

```diff
@@ -37,6 +37,28 @@ module.exports = strapi => {
             taskValue.task
           );
         });
+
+        _.forEach(_.keys(strapi.plugins), pluginName => {
+          const pluginCron = strapi.plugins[pluginName].config.functions.cron;
+
+          if (pluginCron) {
+            _.forEach(_.entries(pluginCron), ([taskExpression, taskValue]) => {
+              if (_.isFunction(taskValue)) {
```

Comment:
In order to avoid duplicate code content, I've wrote this version : 

```
const scheduleTask = (taskExpression, taskValue) => {
        if (_.isFunction(taskValue)) {
          return cron.scheduleJob(taskExpression, taskValue);
        }

        const options = _.get(taskValue, 'options', {});

        cron.scheduleJob(
          {
            rule: taskExpression,
            ...options,
          },
          taskValue.task,
        );
      };

      if (strapi.config.get('server.cron.enabled', false) === true) {
        _.forEach(_.entries(strapi.config.get('functions.cron', {})), ([taskExpression, taskValue]) => {
          scheduleTask(taskExpression, taskValue);
        });

        _.forEach(_.keys(strapi.plugins), pluginName => {
          if (strapi.plugins[pluginName].config &&
            strapi.plugins[pluginName].config.functions &&
            strapi.plugins[pluginName].config.functions.cron) {
            const pluginCron = strapi.plugins[pluginName].config.functions.cron;
            if (pluginCron) {
              _.forEach(_.entries(pluginCron), ([taskExpression, taskValue]) => {
                scheduleTask(taskExpression, taskValue);
              });
            }
          }
        });
```

It's working well on my side.
Thanks for this contribution :) I hope Strapi teams will add this soon !! ❤️

Reply:
Updated also!